lab_3_generate_by_ngrams/main.py

```diff
@@ -76,6 +99,12 @@ def get_token(self, element_id: int) -> Optional[str]:
 
         In case of corrupt input arguments or arguments not included in storage, None is returned
         """
+        if not isinstance(element_id, int) or element_id not in self._storage.values():
+            return None
+
+        inv_storage = {value: key for key, value in self._storage.items()}
```

Comment:
Try to do this without creating a new storage

Reply:
done